lms/djangoapps/discussion/plugins.py

```diff
@@ -18,8 +19,11 @@ class DiscussionTab(EnrolledTab):
     title = ugettext_noop('Discussion')
     priority = None
     view_name = 'discussion.views.forum_form_discussion'
+    fragment_view_name = 'discussion.views.DiscussionBoardFragmentView'
```

Comment:
That is a great question! I originally removed the view_name, but that caused the URL to the discussion view to change. I changed the logic so that it is legal to have both, in which case the legacy view can render itself at its old URL, but it still has access to the fragment view name to render its content.